_posts/2020-04-28-US-RSE-progress.md

```diff
@@ -0,0 +1,81 @@
+---
+layout: post
+title: A year of progress for US-RSE
+tags: [communication]
+---
+
+by Daniel S. Katz, Vanessa Sochat, Charles Ferenbaugh, Jeffrey C. Carver, Sandra Gesing, Ian Cosden, Christina Maimone, and Chris Hill
+
+The US-RSE association was first planned in 2018, but it really became active and started growing in 2019. In the last year or so, we've made a lot of progress and have a number of activities that are in-progress. In this post, we want to report on this progress.
+
+## Accomplishments
+
+**Membership**. In January 2019, we had 10 members, who were all involved in starting up US-RSE. By the end of 2019, this had grown to 259, and is now (at the end of March 2020) 345.
+
+![US-RSE Membership]({{ site.baseurl }}/assets/posts/2020-04-28-us-rse-membership.png "US-RSE Membership")
```

Comment:
The image captions that were in the google doc aren't showing up in the rendered version here.  I'm not sure how difficult it would be to fix this, but wanted to at least point it out.

Reply:
good point - let me try to add them